food-truck.eve.md

```diff
@@ -156,148 +156,104 @@ commit
-- Draw top banner
-- Draw truck name
-- Draw back button
-- Clicking takes you back to the home screen
-- Draw message text form
-- Draw picture icon
-- Clicking lets you choose photo from phone storage or live photo
-- Draw social media icons
-- Facebook
-- Twitter
-- Instagram
-- All off by default
-- Clicking turns on that site for posting
-- If credentials are missing, bring up credential screen
-- Draw time box
-- Says “When?” if no choice has been entered yet
-- Clicking brings up time overlay
-- Draw “Now” toggle
-- Draw time and date selection
-- Draw date icon
-- Clicking opens a calendar
-- Clicking a date chooses that date and closes the calendar
-- Draw time selector
-- Scrolling up counts forward in time
-- Scrolling down counts backwards in time
-- Draw check box button
-- When clicked, apply time choice to post
-- If “Now” is toggled on, choose now and revert to social media screen
-- If “Now” is not toggled on, and both a date and a time have been selected, choose that time and revert to social media screen
-- If “Now” is not toggled on, and either a time or a date is missing, flash box red and do nothing
-- Draw “Post!” button
-- If the message form contains text, at least one social media icon is toggled on, and a time is selected, make button clickable
-- When clicked, if time selected is “Now”, commit the message to social media
-- When clicked, if time selected is a later date, add message to social media queue to be posted at that time
-- Draw social media queue
-- Draw thumbnail photo if there’s a photo on the post
-- Draw preview text
-- Draw scheduled time to post
-- Swipe left to remove from the queue
-- Click to parse post details into the social media form and remove message from the queue
-- Replace any contents that may already be present with contents from queued post
-
-### Truck Settings
-- Draw top banner
-- Draw name
-- Clicking brings up a text form filled with the current truck name
-- Draw back button
-- Clicking returns you to the home screen
-- Draw hero pic
-- Clicking lets you choose photo from URL, phone storage, or live photo
-- Draw description form
-- Draw integration buttons
-- If all credentials are present, icon is lit up
-- Else, icon is grayed out
-- Clicking brings up credentials overlay
-- Draw credential forms
-- Draw accept button
-- Clicking applies entered credentials and reverts to truck settings screen
-
-## Cashier
-
-### Order Screen
-- Draw top banner
-- Draw truck name
-- Draw open/closed icon
-- Clicking brings up open/closed overlay
-- Clicking open sets truck status to open for business and reverts back to order screen
-- Clicking closed sets truck status to closed and reverts back to order screen
-- Draw menu items
-- Draw photo
-- Draw item name
-- Swiping right adds one to the cart
-- Swiping left removes one from the cart
-- Draw pay button
-- Clicking pay button takes you to the Stripe payment page/overlay
+```
+search
+  order = [#order #my-order number items status]
```

Comment:
You should search to make sure the app is on the current page at the top of this block. Look at the homepage for an example.